supervision/detection/utils.py

```diff
@@ -48,6 +48,25 @@ def box_area(box):
     return area_inter / (area_true[:, None] + area_detection - area_inter)
 
 
+def mask_iou_batch(masks_true: np.ndarray, masks_detection: np.ndarray) -> np.ndarray:
```

Comment:
@mayankagarwals let's add unit tests

Reply:
Hey! Yep, waiting for some discussion to happen with respect to API here https://github.com/roboflow/supervision/issues/238 before completing the feature